core/dbt/events/functions.py

```diff
@@ -18,103 +18,146 @@
 metadata_vars: Optional[Dict[str, str]] = None
 
 
-def setup_event_logger(
-    log_path: str,
-    log_format: str,
-    use_colors: bool,
-    debug: bool,
-    log_cache_events: bool,
-    quiet: bool,
-) -> None:
+def setup_event_logger(flags) -> None:
     cleanup_event_logger()
-    make_log_dir_if_missing(log_path)
+    make_log_dir_if_missing(flags.LOG_PATH)
 
     if ENABLE_LEGACY_LOGGER:
-        EVENT_MANAGER.add_logger(_get_logbook_log_config(debug))
-    else:
         EVENT_MANAGER.add_logger(
-            _get_stdout_config(log_format, debug, use_colors, log_cache_events, quiet)
+            _get_logbook_log_config(
+                flags.DEBUG, flags.USE_COLORS, flags.LOG_CACHE_EVENTS, flags.QUIET
+            )
         )
+    else:
+        if flags.LOG_LEVEL != "none":
+            log_format = _line_format_from_str(flags.LOG_FORMAT, LineFormat.PlainText)
```

Comment:
is there a difference between `log_format` and `line_format`? If not, `line_format` would be more intuitive given the naming of `_line_format_from_str` + parameter name of `_get_stdout_config`.

Reply:
I agree, the variable being assigned here should be line_format. There is no difference between the two things, and the difference in naming is unfortunate.